scripts/create_tables.sql

```diff
@@ -1 +1,27 @@
 -- create doctors, patients, and appointments tables
+create table doctors(
+  id serial primary key
+  family_name text,
+   given_name text,
+    gender text,
+     born_on date,
+      specialty text
+    );
+
+create table patients (
+  id serial primary key
+    family_name text,
+     given_name text,
+      gender text,
+       born_on date,
+        insurance text
+);
+
+create table appointments(
+  id serial primary key,
+    date date,
+    time time,
+    comment  text,
+    doctor.id integer references doctors(id)
```

Comment:
This should be named `doctor_id` and there needs to be a comma at the end of this line.